src/config/config.ts

```diff
@@ -0,0 +1,23 @@
+import dotenv from 'dotenv';
+dotenv.config();
+
+export default {
+    api: {
+        port: process.env.PORT
+    },
+    db: {
+        user: process.env.DB_USER,
+        password: process.env.DB_PASSWORD,
+        database: process.env.DB_NAME
+    },
+    mail: {
+        host: process.env.MAIL_HOST,
+        port: parseInt(process.env.MAIL_PORT as string),
```

Comment:
```suggestion
        port: parseInt(process.env.MAIL_PORT as string, 10),
```
Otherwise, 10 might not be radix used.

Reply:
Done